modules/transport-netty4/src/test/java/org/elasticsearch/http/netty4/Netty4HttpServerTransportTests.java

```diff
@@ -568,11 +568,9 @@ public void dispatchBadRequest(final RestChannel channel, final ThreadContext th
             }
         };
         // there's only one netty worker thread that's reused across client requests
-        Settings settings = Settings.builder()
-            .put(Netty4Plugin.WORKER_COUNT.getKey(), 1)
+        Settings settings = createBuilderWithPort().put(Netty4Plugin.WORKER_COUNT.getKey(), 1)
```

Comment:
I think this is the problem. I think the error we're seeing in https://github.com/elastic/elasticsearch/issues/108193 is an exception due to an RST state for the socket.

Reply:
You are right. Funnily, this was the first thing I tried. Not sure why it failed for me before, but I re-tested it now and it passes without exceptions.